cmd/demo/main.go

```diff
@@ -31,7 +32,23 @@ func main() {
 		fmt.Printf("\t%s\n", c)
 	}
 
-	app, err := demo.New(os.Getenv("PGPASSWORD"))
+	// Open postgres connection
+	//
+	dsn := fmt.Sprintf(
+		"postgres://%s:%s@%s:%s/%s",
+		os.Getenv("PGUSER"),
+		os.Getenv("PGPASSWORD"),
+		os.Getenv("PGHOST"),
+		os.Getenv("PGPORT"),
+		os.Getenv("PGDATABASE"),
+	)
+	db, err := database.Open("pgx", dsn)
```

Comment:
OOS, but will this eventually need a way to reopen the connection in case it drops?

Reply:
pgx maintains a pool of actual connections. The handle returned from Open isn't really a connection.